### PR TITLE
feat: allow chose prediction progress bar

### PR DESCRIPTION
**Pull request by Borda**
training already has this option to use `rich` or `tqdm` if set as false and some notebooks do not like rich or for a huge number of small predictions the progress bar is quite spammy

----
:books: Documentation preview :books:: https://pytorch-tabular--305.org.readthedocs.build/en/305/

### CODE REVIEW

**Comment by manujosephv** [comment]
This was much needed. thanks @Borda. But I also see the version bumped.. We are still at 1.0.2. Can you revert that? rest of it looks good to merge

**Comment by Borda** [comment]
> But I also see the version bumped.. We are still at 1.0.2. Can you revert that? rest of it looks good to merge

Yes I can, I just needed it as I have to install this version from branch and don't break env... But anyway I would recommend to use dev when new feature were udded until they are released so users installing from source won't be confused =)

File: setup.py [hunk 1]
```diff
@@ -62,6 +62,6 @@ def read_requirements(thelibFolder, filename):
     test_suite="tests",
     tests_require=test_requirements,
     url="https://github.com/manujosephv/pytorch_tabular",
-    version="1.0.2",
+    version="1.0.3.dev1",
```
Comment (Borda) [inline setup.py:65]:
```suggestion
    version="1.0.2",
```

File: src/pytorch_tabular/__init__.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 __author__ = """Manu Joseph"""
 __email__ = "manujosephv@gmail.com"
-__version__ = "1.0.2"
+__version__ = "1.0.3.dev1"
```
Comment (Borda) [inline src/pytorch_tabular/__init__.py:5]:
```suggestion
__version__ = "1.0.2"
```

**Review by manujosephv — APPROVED** [review]

**Comment by kaushikb11** [comment]
Rich Progress Bar ftw!

**Pull request merged**